mkt/reviewers/urls.py

```diff
@@ -71,6 +72,8 @@
     url(r'reviewers/', include(reviewers_router.urls)),
     url('^reviewers/search', views.ReviewersSearchView.as_view(),
         name='reviewers-search-api'),
+    url('^reviewers/sites/search', WebsiteSearchView.as_view(),
```

Comment:
there's already a `ReviewersSearchView` used in the line previous, want me to rename it to `ReviewersWebsiteSearchView`?